Update use-http-context.md (#31420)
dotnet/AspNetCore.Docs: aspnetcore/fundamentals/use-http-context.md

[!code-csharp[](use-http-context/samples/Program.cs?name=snippet_RequestHeaders&highlight=6-7)]

> For information on efficiently storing and fetching header strings, see [A brief look at StringValues](https://andrewlock.net/a-brief-look-at-stringvalues/).

Author: ```suggestion
For information on efficiently handling headers that appear more than once, see [A brief look at StringValues](https://andrewlock.net/a-brief-look-at-stringvalues/).
```
